inc/class-shortcode-ui.php

```diff
@@ -81,6 +82,44 @@ private function setup_actions() {
 	}
 
 	/**
+	 * Display an admin notice on activation.
+	 *
+	 * If no shortcodes with Shortcake UI are registered, this Will display a link to the plugin's wiki for examples.
+	 * If there are already plugins with UI registered, will just display a success message.
+	 *
+	 * @return void
+	 */
+	public function action_admin_notices() {
+		if ( ! get_option( 'shortcode_ui_activation_notice' ) ) {
+			return;
+		}
+
+		if ( ! $this->has_shortcodes() ) {
+			echo '<div class="notice notice-warning is-dismissable"><p>' .
+				sprintf(
+					wp_kses(
+						/* Translators: link to plugin wiki page with examples of shortcodes supporting Shortcake UI */
+						__( 'The Shortcode UI plugin will not do anything unless UI is registered for shortcodes through a theme or plugins. For examples, see <a href="%s" target="_blank">here</a>.', 'shortcode-ui' ),
+						array(
+							'a' => array(
+								'href' => array(),
+								'target' => array(),
+							),
+						)
+					),
+					'https://github.com/wp-shortcake/shortcake/wiki/Shortcode-UI-Examples'
+				) .
+				'</p></div>' . "\n";
+		} else {
+			echo '<div class="notice notice-info is-dismissable"><p>' .
+				esc_html__( 'That\'s all! Try out the shortcode UI through the "Add Post element" button in the post edit screen.', 'shortcode-ui' ) .
```

Comment:
I'm not so sure about 'Thats all'. I'm concerned it won't be immediately clear what its referring to. EG if the tab has been left open and the user comes back to it, the context of having just clicked activate is lost. 

How about `Shortcode UI installed. Try it out through the "Add Post element" button in the post edit screen."

'